debezium-quarkus-outbox-common/deployment/src/main/java/io/quarkus/kafka/client/deployment/RedpandaKafkaContainer.java

```diff
@@ -0,0 +1,135 @@
+/*
```

Comment:
hack, will remove it when changed will be pushed and release inside Quarkus
Keep pr in draft until done

Reply:
https://github.com/quarkusio/quarkus/pull/43582